### ISSUE TITLE
Repository.open returns empty object

### ISSUE THREAD

**Issue by max-mykhailenko** [nodegit/nodegit#412]
Try to use your `status` example and it doesn't work
https://github.com/nodegit/nodegit/blob/master/examples/status.js#L6-L8
Logging `repo` shows empty object.

**Comment by tbranyen** [comment]
Logging out NodeGit objects will make them appear empty.  This isn't a fault with NodeGit or your program, it's just how things get logged.  If you were to log `repo.__proto__` you'd see the prototype object looked up.

**Comment by max-mykhailenko** [comment]
Any log after https://github.com/nodegit/nodegit/blob/master/examples/status.js#L8 (in git status callback) doesn't run.

**Comment by tbranyen** [comment]
Probably due to an error, if you want to see if an error is logged, change that line to:

``` javascript
    repo.getStatus().catch(console.log.bind(console)).then(function(statuses) {
```

**Comment by max-mykhailenko** [comment]
Console is empty anyway

**Comment by max-mykhailenko** [comment]
Find error in catch of `getStatus`
```
{ '0': [TypeError: Object #<Repository> has no method 'getStatus'] }
```

**Comment by johnhaley81** [comment]
Is this on `master` or `0.2.7`?

**Comment by max-mykhailenko** [comment]
After `npm install nodegit --save-dev` I got `"nodegit": "^0.2.7"` in my `package.json`

**Comment by johnhaley81** [comment]
`repo.getStatus` is only on master.

**Comment by maxkorp** [comment]
Yeah, a LOT of the functionality has been implemented on master, ahead of 0.2.7. We're pushing forward with an 0.3 release as quickly as possible, to get that stuff out in the wild where its safe to use.
